ddtrace/appsec/_iast/_handlers.py

```diff
@@ -556,3 +556,14 @@ async def _iast_instrument_starlette_request_body(wrapped, instance, args, kwarg
     return taint_pyobject(
         result, source_name=origin_to_str(OriginType.PATH), source_value=result, source_origin=OriginType.BODY
     )
+
+
+def _iast_instrument_starlette_scope(scope, route):
```

Comment:
`route` isn't used?

Reply:
I use it in the "main" PR: https://github.com/DataDog/dd-trace-py/pull/13347. I'm splitting the that PR because it's too big